config/index.js

```diff
@@ -34,7 +34,8 @@ const config = {
     api: {
       bodySizeLimit: process.env.API_BODY_SIZE_LIMIT,
       parameterLimit: process.env.API_PARAMETER_LIMIT,
-      port: process.env.PORT
+      port: process.env.PORT,
+      url: process.env.URL
```

Comment:
This is fine but what do you think if to be a little more organized we add a new property that is especially for the Weets API? For example WeetApi, what do you think?

Reply:
off course, I like the idea about name for the WeetApi. I change.